R/Google.R

```diff
@@ -0,0 +1,191 @@
+#' R6 Class containing specific attributes and methods for Google data
+#'
+#' @description Google specific information for downloading, cleaning
+#'  and processing covid-19 region data for an example Country.
+#'
+#' @source \url{https://github.com/GoogleCloudPlatform/covid-19-open-data}
+#' @export
+#' @concept dataset
+#' @examples
+#' \dontrun{
+#' region <- Google$new(verbose = TRUE, steps = TRUE, get = TRUE)
```

Comment:
Add an example of using for each level of data as we expect users to use this class on its own at least initially. Do the same for JHU.